policyengine_us/parameters/gov/states/ky/tax/income/credits/personal/amount/military.yaml

```diff
@@ -0,0 +1,13 @@
+description: Kentucky allows for this personal tax credits amount for filers with military service.
+values:
+  2021-01-01: 20
+metadata:
+  unit: year
+  label: Kentucky personal tax credits amount for military filers
+  reference:
+    - title: Schedule ITC Section B 2022
+      href: https://revenue.ky.gov/Forms/Schedule%20ITC%202022.pdf#page=2
+    - title: Schedule ITC Section B 2021
+      href: https://revenue.ky.gov/Forms/Schedule%20ITC-2021.pdf#page=2
+    - title: Kentucky Revised Statutes Chapter 141.020
+      href: https://apps.legislature.ky.gov/law/statutes/statute.aspx?id=53500
```

Comment:
create a non_refundable.yaml file or refundable.yaml - depending on whether it is a refundable or non-refundable credit

Reply:
also add to .py file